assign4.1WithPromise.js

```diff
@@ -0,0 +1,44 @@
+var fs = require('fs');
+var file1Content = ` file 1 content goes 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 
+  1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 
+  1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 `;
+fs.writeFile("file1.txt",file1Content,function(err,data) {
+  if(err) throw err;
+  else
+  console.log("file 1 content");
+});
```

Comment:
this code gets executed asynchronously. File write should also be promisified to make sure order of execution is correct